config/base_config.go

```diff
@@ -64,7 +83,7 @@ func (c *BaseConfig) startConfigCenter() error {
 	if c.prepareEnvironment() != nil {
 		return perrors.WithMessagef(err, "start config center error!")
 	}
-	//c.fresh()
+	// c.fresh()
 	return err
```

Comment:
why delete fresh?

Reply:
It was deleted by you and I don't know why too.  I just format the comment:
![image](https://user-images.githubusercontent.com/9923838/84654927-9f16df00-af42-11ea-9a66-8a2e620363b8.png)

Reply:
OK